module/zfs/dmu_send.c

```diff
@@ -1527,13 +1536,14 @@ dmu_recv_begin_sync(void *arg, dmu_tx_t *tx)
 	const char *tofs = drba->drba_cookie->drc_tofs;
 	dsl_dataset_t *ds, *newds;
 	uint64_t dsobj;
+	int flags = DS_HOLD_FLAG_DECRYPT;
```

Comment:
I only called it `dsflags` there because `flags` was already being used. Would you like me to change this everywhere?